WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -21,7 +22,10 @@ - (BOOL)fb_tapWithError:(NSError **)error
   if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"13.0")) {
     // Tap coordinates calculation issues have been fixed
     // for different device orientations since Xcode 11
+    [self fb_waitUntilFrameIsStable];
```

Comment:
Please add an explanation about why this workaround is necessary (a link to this PR could be enough)